app/BUILD.bazel

```diff
@@ -95,7 +95,7 @@ VIEW_MODEL_FILES = [
     "src/main/java/org/oppia/app/options/OptionsAppLanguageViewModel.kt",
     "src/main/java/org/oppia/app/options/OptionsAudioLanguageViewModel.kt",
     "src/main/java/org/oppia/app/options/OptionsItemViewModel.kt",
-    "src/main/java/org/oppia/app/options/OptionsStoryTextViewViewModel.kt",
+    #"src/main/java/org/oppia/app/options/OptionsStoryTextViewViewModel.kt",
```

Comment:
Either remove the comment or remove the line--never check in commented code.

Reply:
Done